chat-summary/summarize.gpt

```diff
@@ -1,3 +1,5 @@
+Type: context
```

Comment:
as mentioned below, i think this is not a context tool

Reply:
Correct, not context.  This is regular tool called by python